dlup/utils/pyvips.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
```

Comment:
do we really need these functions? What if we let the user take care of this?

Reply:
Well, yes, but they are pain to remember.